google/auth/transport/grpc.py

```diff
@@ -132,7 +132,7 @@ def secure_authorized_channel(
 
         # Create a channel.
         channel = google.auth.transport.grpc.secure_authorized_channel(
-            credentials, regular_endpoint, request,
+            credentials, request, regular_endpoint,
```

Comment:
why use posisional args again here?  woudn't it be clearer using the names itself for clarity   `request=request, regular_endpoint=`...)

Reply:
I don't feel particularly strongly about this, but switching to keyword arguments would go against the style of the rest of the codebase.

Reply:
sure, but using  incorrect positional led to filing this bug.   apart from styleguide stuff, the better way to think about it is what would be the best for the customer/reader